app/models/vetting_record.rb

```diff
@@ -1,5 +1,8 @@
 class VettingRecord < ActiveRecord::Base
   belongs_to :user
   validates :skill_proficiency, :experience, :confidence, presence: true
-  scope :find_by_user_id, -> (user_id) { find_by(user_id: user_id) }
+
+  def self.find_by_user_id(user_id)
+    VettingRecord.find_by(user_id: user_id)
+  end
```

Comment:
I see you're advocating for class methods. That's good! We'll need to chat more about it and see the major differences and pros of scopes.